login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { ChatBubbleOutline } from '@mui/icons-material';
+import { ContactSupportScreenBase } from './ContactSupportScreenBase';
+import { ContactSupportScreenProps } from './types';
+import { useAuthContext } from '../../contexts';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * Component renders a screen with contact information for support with the application.
+ * Contact information is pulled from the context passed into the workflow.
+ *
+ * @param icon the icon to display in the header
+ * @param emailSupportTitle text to display as the contact support title
+ * @param emailSupportContent content to display as the contact support content
+ * @param phoneSupportTitle text to display as the phone support title
+ * @param phoneSupportContent content to display as the phone support content
+ * @param contactEmail to display the contact email
+ * @param contactPhone to display contact phone number
+ * @param dismissButtonLabel to display label for the button
+ * @param onDismiss function to call when user clicks button
+ * @param WorkflowCardBaseProps props that will be passed to the WorkflowCard component
+ * @param WorkflowCardHeaderProps props that will be passed to the WorkflowCardHeader component
+ * @param WorkflowCardInstructionProps props that will be passed to the WorkflowCardInstructions component
+ * @param WorkflowCardActionsProps props that will be passed to the WorkflowCardActions component
+ *
+ * @category Component
+ */
+
+export const ContactSupportScreen: React.FC<ContactSupportScreenProps> = (props) => {
+    const { t } = useTranslation();
+    const { navigate, routeConfig } = useAuthContext();
+
+    const {
+        icon = <ChatBubbleOutline color={'primary'} sx={{ fontSize: 70 }} />,
+        emailSupportTitle = t('bluiAuth:CONTACT_SUPPORT.GENERAL_QUESTIONS'),
+        emailSupportContent,
```

Comment:
the emailSupportContent should have translation

Reply:
Fixed https://github.com/etn-ccis/blui-react-workflows/pull/497